### PR TITLE
246 address to geolocation

### PR DESCRIPTION
**Pull request by BoyPersoon**
I Injected the GOOGLE_API_KEY through the service.yml file. However I haven't been able to figure out how to inject it during testing. The tests pass when I add the key manually as argument. suggesting that the service behaves as expected. Invoking it seems to be a problem.

Also I added a Coordinates class to encapsulate geolocation data.

### CODE REVIEW

File: src/Infrastructure/Dto/Coordinates/Coordinates.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+declare(strict_types=1);
+
+namespace App\Infrastructure\Dto\Coordinates;
+
+class Coordinates
+{
+    /**
+     * @var float
+     */
+    private $latitude;
```
Comment (bartrail) [inline src/Infrastructure/Dto/Coordinates/Coordinates.php]:
```suggestion
    private float $latitude;
```

we are using php7.4 already, so you can define typed properties and remove the annotation :)

File: src/Infrastructure/Dto/Coordinates/Coordinates.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    /**
+     * @var float
+     */
+    private $longitude;
+
+    /**
+     * @param float $latitude
+     * @param float $longitude
+     */
+    public function __construct($latitude, $longitude)
```
Comment (bartrail) [inline src/Infrastructure/Dto/Coordinates/Coordinates.php]:
```suggestion
    public function __construct(float $latitude, float $longitude)
```
here as well

File: .env [hunk 3]
```diff
@@ -35,3 +35,6 @@ EMAIL_CONTACT=contact@print4health.org
 ###> sentry/sentry-symfony ###
 SENTRY_DSN=
 ###< sentry/sentry-symfony ###
+###> google/geocoding service ###
+GOOGLE_MAPS_API_KEY=
```
Comment (bartrail) [inline .env:39]:
```suggestion
GOOGLE_MAPS_API_KEY=''
```
not sure about this but I think it should be an empty string?

**Comment by gherkins** [comment]
> I Injected the GOOGLE_API_KEY through the service.yml file. However I haven't been able to figure out how to inject it during testing. The tests pass when I add the key manually as argument. suggesting that the service behaves as expected. Invoking it seems to be a problem.

did you add the key to .env.test as that's the env file used for the phpunit tests...?

**Review by bartrail — DISMISSED** [review]

**Review by bartrail — APPROVED** [review]

**Pull request merged**